cmd/prometheus/main.go

```diff
@@ -288,6 +292,17 @@ func main() {
 
 	var g group.Group
 	{
+		g.Add(
+			func() error {
+				return discoveryManager.Run()
+			},
+			func(err error) {
+				level.Info(logger).Log("msg", "Stopping target manager...")
```

Comment:
Should this be "discovery manager" rather than "target manager" ?

Reply:
thanks. noted will change that when I do the comments